### PR TITLE
framework/task_manager : Move task manager creation to preapp

### PR DESCRIPTION
**Pull request by jeongchanKim**
The creation of task manager is not the kernel role, so moved to preapp.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [136500bf3d79d20f476c4797cc6d54177bdea2b8] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [136500bf3d79d20f476c4797cc6d54177bdea2b8] - Code Rule Check OK.

File: apps/system/init/init.c [hunk 1]
```diff
@@ -36,6 +36,10 @@
 #ifdef CONFIG_SYSTEM_INFORMATION
 #include <apps/system/sysinfo.h>
 #endif
+#ifdef CONFIG_TASK_MANAGER
+#include <tinyara/init.h>
+#include <tinyara/task_manager_drv.h>
```
Comment (sunghan-chang) [inline apps/system/init/init.c]: Unnecessary

File: apps/system/init/init.c [hunk 2]
```diff
@@ -131,6 +135,17 @@ int preapp_start(int argc, char *argv[])
 	tash_register_cmds();
 #endif
 
+#ifdef CONFIG_TASK_MANAGER
+#define TASKMGR_STACK_SIZE 2048
+#define TASKMGR_PRIORITY 200
+	printf("Starting task manager\n");
+
+	pid = task_create("task_manager", TASKMGR_PRIORITY, TASKMGR_STACK_SIZE, task_manager, (FAR char *const *)NULL);
```
Comment (sunghan-chang) [inline apps/system/init/init.c]: To use `pid`, let's add a conditional at line 107.

**Comment by sunghan-chang** [comment]
And this modification is not in framework folder. Let's remove `framework/'

**Comment by seinfra** [comment]
Target : [70156386d90b9a03ce63b03956cfc5306ee7dbcd] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [70156386d90b9a03ce63b03956cfc5306ee7dbcd] - Code Rule Check OK.

**Pull request closed without merging**